### PR TITLE
Email registration

### PR DESCRIPTION
**Pull request by allenaerostar**
Email registration with verification and resend verification views completed along with test cases.

### CODE REVIEW

File: dietstory-routing/registration/views.py [hunk 1]
```diff
@@ -32,6 +34,7 @@ def post(self, request, *args, **kwargs):
                     account = Accounts(name=username, password=password, email=email, birthday=birthday)
                     account.save()
 
+                    send_email(account.email, account_activation_token.make_token(account))
```
Comment (BenjixD) [inline dietstory-routing/registration/views.py]: consider try/catch this line

File: dietstory-routing/dietstory-routing/dietstory_secret_config_template.json [hunk 2]
```diff
@@ -10,6 +10,12 @@
 	        "PORT": "<PORT>"
 	    }
 	},
+	"EMAIL_BACKEND": "<DJANGO_EMAIL_BACKEND>",
+    "EMAIL_HOST": "<EMAIL_HOST>",
+    "EMAIL_PORT": "<PORT>",
+    "EMAIL_HOST_USER": "<EMAIL>",
+    "EMAIL_HOST_PASSWORD": "<PASSWORD>",
+    "EMAIL_USE_TLS": "<True/False>",
 	"TEST_DATABASES": {
```
[the remark this answers is not in the document]
Comment (allenaerostar) [reply dietstory-routing/dietstory-routing/dietstory_secret_config_template.json]: Can't seem to find the issue on my own local...

File: dietstory-routing/registration/models.py [hunk 3]
```diff
@@ -8,20 +8,31 @@
 from django.db import models
 from django.core.validators import MinValueValidator, MaxValueValidator
 
+dateFieldDefault = "0000-00-00"
+dateTimeFieldDefault = "0000-00-00 00:00:00"
+
```
[the remark this answers is not in the document]
Comment (allenaerostar) [reply dietstory-routing/registration/models.py]: Good point, fixed this.

File: dietstory-routing/registration/models.py [hunk 4]
```diff
@@ -8,20 +8,31 @@
...
 
 class ZeroDateTimeField(models.DateTimeField):
     def get_db_prep_value(self, value, connection, prepared=False):
-        value = super(ZeroDateTimeField, self).get_db_prep_value(value, connection, prepared)
+        if value == dateTimeFieldDefault:
+            pass
```
[the remark this answers is not in the document]
Comment (allenaerostar) [reply dietstory-routing/registration/models.py]: Woops haha

**Pull request merged**